web/libs/core/project.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "core",
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "libs/core/src",
+  "projectType": "library",
+  "tags": [],
+  "targets": {
+    "lint": {
+      "executor": "@nx/eslint:lint",
```

Comment:
I see eslint here - was that intended? would be awesome to just use biome through out if possible